tests/drivers/build_all/gpio/adc_ads1145s0x_gpio.overlay

```diff
@@ -39,8 +39,8 @@
 				drdy-gpios = <&test_gpio 0 0>;
 				start-sync-gpios = <&test_gpio 0 0>;
 
-				test_spi_ads114s08_gpio: ads114s0x_gpio {
-					compatible = "ti,ads114s0x-gpio";
+				test_spi_ads114s08_gpio: ads1x4s0x_gpio {
```

Comment:
Could you please add one instance of each binding to the build_all tests?

Reply:
Done.

Reply:
done